catalog/core/catalog-core-camelcomponent/pom.xml

```diff
@@ -170,6 +170,35 @@
                     </instructions>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>1.4.1</version>
+                <dependencies>
```

Comment:
✏️ Move this dependency part in the parent pom plugins management section where you can define this plugin with its version and dependencies. This way, you won't have to repeat it in every pom files.

Reply:
I was going to say there isn't much advantage to doing that, but it is eight lines that can be removed from any module's pom that needs to specify its own threshold. Good call.